### DISCUSSION TITLE
FERTILE VOID: Studio Quantum Showcase at Haus der Kulturen der Welt [Berlin, DE] (1-11-2024)

### DISCUSSION THREAD

**Discussion by studiofomalhaut** [Quantumland-art/community#77] [Art]
_[FERTILE VOID: Studio Quantum Showcase](https://www.hkw.de/en/programme/fertile-void/studio-quantum-showcase#main)_

Haus der Kulturen der Welt in Berlin presents this exciting showcase with works by Amy Karle, kennedy+swan, Edy Fung, and Ìfẹ́olúwa Ọ̀ṣúnkọ́yà. The exhibition "seeks to interrogate quantum science as an approximation, scale, translation, and condition of a potentially multiplicitous physical reality, one that understands the combination of energy and matter as more than the sum of its parts. In this spirit, four artists of the Studio Quantum residency 2023 have created works that delve into quantum concepts from four distinct fields, each reaching towards a particular cosmology of technology, science, and embodiment."

The four artistic engagements with quantum are inserted into the current HKW exhibition Forgive us our Tresspasses / Vergib uns unsere Schuld. The presence of the works that translate, aestheticize, and narrativize quantum physics also fundamentally question the exceptionality of trespassing, implying that from a quantum perspective, everything is acting in convergence at all times, and not all is what it may seem. The artworks thus transgress binary notions of inclusion and exclusion, and indeed, of physical space, as they work with the sonic, the spiritual, and the virtual as technologies of a quantum realm."

_**Studio Quantum**_ is an international events and artist-in-residence programme from the Goethe-Institut, exploring emerging quantum technologies through the lens of art. Artistic explorations of the synergies between art and technology as well as impressions and learnings from the international residencies are an integral part of the showcased works. 

_**Fertile Void**_ is a cooperation between Haus der Kulturen der Welt (HKW) and the Goethe-Institut Irland / Studio Quantum, with contributions from the JUNGE AKADEMIE of the Akademie der Künste,  Schering Stiftung, Universität der Künste, Berlin and Trinity College Dublin. The project is part of Berlin Science Week. With thanks to Quantum Delta, ZKM, ITAS (KIT), Science Gallery London, King's College and Zeitgeist Irland 24, an initiative of Culture Ireland and the Embassy of Ireland in Germany.

**Comment by Itaborala** [comment]
Very nice!

**Comment by omarcostahamido** [comment]
thank you @studiofomalhaut! I just realized this is integrated in the berlin science week as well!
https://berlinscienceweek.com/programme/fertile-void-installations-day-1

**Comment by studiofomalhaut** [comment]
@omarcostahamido Oh, yes, that's great!
